tt_metal/hw/ckernels/grayskull/metal/llk_api/llk_sfpu/ckernel_sfpu_floor.h

```diff
@@ -0,0 +1,29 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "ckernel.h"
+#include "ckernel_defs.h"
+#include "sfpi.h"
+#include "noc_nonblocking_api.h"
+
+using namespace sfpi;
+
+namespace ckernel {
+namespace sfpu {
+
+//TODO: Implementation pending for GS
+template <bool APPROXIMATION_MODE, int ITERATIONS = 4>
+inline void calculate_floor()
+{
```

Comment:
Add `static_assert("GS not supported for this op");` with link to the issue number

Reply:
Addressed @rtawfik01